webpack.common.config.js

```diff
@@ -17,15 +17,19 @@ let jsOutput = '[name].bundle.js';
 let styleOutput = '[name].bundle.css';
 let mediaOutput = '[name].[ext]';
 let devtool = 'cheap-module-inline-source-map';
-let buildScript = 'python run.py dev';
+// @@@ Sassdoc does not properly update the st_mtime (modified time) on
+// re-generated files, so we empty the output styleguide/ dir before copying
+// changed files.
+// See https://github.com/oddbird/oddsite/issues/55
+let buildScript = 'rm -rf ./dev-output/styleguide/* && python run.py dev';
```

Comment:
@wlonk I'm not thrilled with this, but I went this route because it felt weird for Sassdoc to be responsible to reach out to the `dev-output` directory to remove files. But this also feels pretty hacky, so 🤷